distribution/changelog.txt

```diff
@@ -28,6 +28,7 @@
 - Fix: [#19950] Mine train block brake supports drawn incorrectly.
 - Fix: [#19987] [Plugin] ‘SetCheatAction’ has wrong ID in plugin API.
 - Fix: [#19991] Crash using cut-away view.
+- Fix: [#16791] Rotodrop top piece replicates when walls are placed around it and clearance checks are disabled.
```

Comment:
we keep this in order of the issue so this needs to move up to L17. Should also mention that this is an "(original bug)" see below how that is normally marked.

Reply:
Moved, thanks for letting me know